src/Avalonia.Base/Input/KeyGesture.cs

```diff
@@ -143,7 +144,7 @@ static void Plus(StringBuilder s)
         // TODO: Move that to external key parser
         private static Key ParseKey(string key)
         {
-            if (s_keySynonyms.TryGetValue(key.ToLower(), out Key rv))
+            if (s_keySynonyms.TryGetValue(key.ToLower(CultureInfo.InvariantCulture), out Key rv))
```

Comment:
Note for the future there is a `.ToLowerInvariant()` that should be used in places like this.